src/mpyl/utilities/subprocess/__init__.py

```diff
@@ -9,7 +9,10 @@
 
 
 def custom_check_output(
-    logger: Logger, command: Union[str, list[str]], capture_stdout: bool = False
+    logger: Logger,
+    command: Union[str, list[str]],
+    capture_stdout: bool = False,
+    use_print: bool = False,
```

Comment:
Maybe `stream_output` instead of print? but both are fine

Reply:
but both print and log can be / are streams?

Reply:
fair enough